data/json/items/gun/357.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "p226_357sig",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "SIG P226",
+    "description": "A SIG Sauer P226 chambered for .357 SIG.  The P226 is a DA/SA, short-recoil operated semi-automatic pistol.",
+    "weight": 800,
+    "volume": 2,
+    "price": 50000,
+    "to_hit": 1,
```

Comment:
Most of guns I reviewed for comparison have ```to_hit: -1``` and I think this gun should be the same.